src/porepy/numerics/fv/mpfa.py

```diff
@@ -74,6 +74,8 @@ def discretize(self, g, data):
         data (dict): For entries, see above.
         faces (np.ndarray): optional. Defines active faces.
         """
+        tol = data.get("tol", 1e-5)
```

Comment:
This cannot be called tol (and the same then applies to rt0 above); the name is too general. Geometric tolerance is not sufficiently specific. Perhaps deviation_from_plane_tol? It is long, but clear.

Also, the option should be documented. Same with RT0, and presumably mvem below.

Reply:
ok, thanks for the suggestion